### ISSUE TITLE
Installation trouble

### ISSUE THREAD

**Issue by rdrey** [keredson/wordninja#1]
Hey, found this package via the SO answer.

It doesn't seem like the `.tar.gz` file is pip installed into the right location:

```
import wordninja

---------------------------------------------------------------------------
IOError                                   Traceback (most recent call last)
<ipython-input-3-b786e3290074> in <module>()
----> 1 import wordninja

/home/rainer/.virtualenvs/ml/local/lib/python2.7/site-packages/wordninja.py in <module>()
     14 
     15 # Build a cost dictionary, assuming Zipf's law and cost = -math.log(probability).
---> 16 with gzip.open(os.path.join(os.path.dirname(os.path.abspath(__file__)),'wordninja_words.txt.gz')) as f:
     17   words = f.read().decode().split()
     18 _wordcost = dict((k, log((i+1)*log(len(words)))) for i,k in enumerate(words))

/usr/lib/python2.7/gzip.pyc in open(filename, mode, compresslevel)
     32 
     33     """
---> 34     return GzipFile(filename, mode, compresslevel)
     35 
     36 class GzipFile(io.BufferedIOBase):

/usr/lib/python2.7/gzip.pyc in __init__(self, filename, mode, compresslevel, fileobj, mtime)
     92             mode += 'b'
     93         if fileobj is None:
---> 94             fileobj = self.myfileobj = __builtin__.open(filename, mode or 'rb')
     95         if filename is None:
     96             # Issue #13781: os.fdopen() creates a fileobj with a bogus name

IOError: [Errno 2] No such file or directory: '/home/rainer/.virtualenvs/ml/local/lib/python2.7/site-packages/wordninja_words.txt.gz'
```

Instead it seems to be in the root of my virtualenv:

```
rainer@rainer-Galago-Pro ~/.v/ml> ls
bin/  include/  lib/  local/  pip-selfcheck.json  share/  wordninja_words.txt.gz
rainer@rainer-Galago-Pro ~/.v/ml> pwd
/home/rainer/.virtualenvs/ml
```

**Comment by keredson** [comment]
interesting.  i was able to dup w/:
```
$ cd /tmp
$ mkdir testve
$ cd testve
$ virtualenv my_project
Running virtualenv with interpreter /usr/bin/python2
New python executable in /tmp/testve/my_project/bin/python2
Also creating executable in /tmp/testve/my_project/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.
$ source my_project/bin/activate
(my_project) $ pip install wordninja
Collecting wordninja
  Downloading wordninja-0.1.2.tar.gz (538kB)
    100% |████████████████████████████████| 542kB 2.1MB/s 
Building wheels for collected packages: wordninja
  Running setup.py bdist_wheel for wordninja ... done
  Stored in directory: /home/derek/.cache/pip/wheels/f9/a7/d7/02ec9ca49189bca2a1e09d6ae20f1fdab03883a952f1397425
Successfully built wordninja
Installing collected packages: wordninja
Successfully installed wordninja-0.1.2
(my_project) $ python
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import wordninja
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/testve/my_project/local/lib/python2.7/site-packages/wordninja.py", line 16, in <module>
    with gzip.open(os.path.join(os.path.dirname(os.path.abspath(__file__)),'wordninja_words.txt.gz')) as f:
  File "/usr/lib/python2.7/gzip.py", line 34, in open
    return GzipFile(filename, mode, compresslevel)
  File "/usr/lib/python2.7/gzip.py", line 94, in __init__
    fileobj = self.myfileobj = __builtin__.open(filename, mode or 'rb')
IOError: [Errno 2] No such file or directory: '/tmp/testve/my_project/local/lib/python2.7/site-packages/wordninja_words.txt.gz'
>>> 
(my_project) $ find . | grep wordninja_words.txt.gz
./my_project/wordninja_words.txt.gz
```

**Comment by keredson** [comment]
all right, this should be fixed in 0.1.3 though the dark arts and horrors of `distutils`.  :)
